### PR TITLE
DOC: Fix some Sphinx warnings

### PR DESCRIPTION
**Pull request by fepegar**
Many warnings are printed out when building the Sphinx docs, as shown in
- #231

This PR fixes two of the three issues.

Before:

Capture2

After:

Capture

### CODE REVIEW

File: docs/Makefile [hunk 1]
```diff
@@ -18,4 +18,5 @@ help:
 # "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
 %: Makefile
 	cp ../CHANGELOG.md source/
+	cp CONTRIBUTING.md source/
```
Comment (mebristo) [inline docs/Makefile:21]: Does this file currently exist? Should this be `../CONTRIBUTING.md` if we are moving it to the root? Also I don't believe we currently have this file at the root level, so would you alsoneed to move that as part of this PR?

Comment (fepegar) [reply docs/Makefile:21]: > Does this file currently exist?

Yes, it's in `hi-ml/docs/`.

> Should this be ../CONTRIBUTING.md if we are moving it to the root?

If we do, yes. But I wanted to ask your opinion before moving it. Shall I?

Comment (mebristo) [reply docs/Makefile:21]: I see no reason not to. @ant0nsc?

**Review by ant0nsc — APPROVED** [review]

File: docs/Makefile (hunk 1 above)
Comment (ant0nsc) [reply docs/Makefile:21]: fine to move, sure.

**Pull request merged**